packages/database-legacy/src/migrations/125_alter_activity_table.ts

```diff
@@ -0,0 +1,28 @@
+/*
+                      ====== IMPORTANT NOTICE ======
+
+DO NOT EDIT OR RENAME THIS FILE
+
+This is a migration file. Once created the file should not be renamed or edited,
+because migrations are only run once on the production server. 
+
+If you find that something was incorrectly set up in the `up` function you
+should create a new migration file that fixes the issue.
+
+*/
+
+import { Knex } from 'knex'
+
+export async function up(knex: Knex) {
+  await knex.schema.alterTable('activity', function (table) {
+    table.integer('start').notNullable().defaultTo(0)
+    table.integer('end').notNullable().defaultTo(0)
```

Comment:
I think we should get rid of this default

Reply:
If there is already data in the table migration will throw so we can either clear the table or provide default. IMHO default is safer.